app/src/test/java/com/rickyhu/hushkeyboard/ui/SettingsScreenUiTest.kt

```diff
@@ -22,6 +24,8 @@ import org.robolectric.RobolectricTestRunner
 class SettingsScreenUiTest {
     @get:Rule
     val composeTestRule = createComposeRule()
+    private var addSpaceAfterNotationSwitchValue = true
+    private var vibrateOnTapSwitchValue = true
```

Comment:
Please help me add a black line to separate `composeTestRule` and other data members.

Thanks!

```suggestion

    private var addSpaceAfterNotationSwitchValue = true
    private var vibrateOnTapSwitchValue = true
```